### ISSUE TITLE
Vainglory MMR findings

### ISSUE THREAD

**Issue by schneefux** [vainglorygame/vainglorygame.github.io#3]
Quoting myself from Discord:

Digging the API I realized something that must share. Vainglory balances itself.
Over a long period of time, you will have a win rate of 50% guaranteed by the matchmaker. 
Now take Petal as an example for a hero that is "overpowered". Overpowered means that you perform better with the hero than your skill level is. Let's define your skill level as the average of your performance with the most OP and the most UP hero.
If you play a hero that is OP for you, you will get matched against better enemies. Matchmaker adapted your MMR to fit your actual skill + the overpowerness of the hero you play.
You are now at 50% win rate again. Even though the hero you play is better, your enemies are too. 
Your artificially buffed MMR won't stay forever, because as soon as you pick a regular or underpowered hero, your effectual skill is lower than your MMR, you will lose and MMR will drop.
Now, the question is, why do heroes have different win rates?
I guess the sample size is too small. Someone plays Petal, moves up, plays a regular hero (say SAW), moves down. In consequences, Petal's win rate increased and SAW's decreased.
Why can't I analyze hero/build win rates globally?
Easy: every tier has builds and comps that are equally shit on both sides. It is not possible to have a "best" build, because if your build is better, your skill level is higher and your opponents are better. Plus, you only matter ⅓rd (1/6th).
How can I still find out what the best builds are?
I think I found the solution for that already. Although you have a 50% chance of winning, your team as a whole has 50% too and your team isn't equally skilled. I already trained a regression that could calculate your impact on the game to 75% accuracy. 
What I should try now is not to correlate hero/build to win/loss, but to impact. In simple terms, who carries the most. That way, I can identify players that are increasing their MMR and essentially approximate the difference between actual skill and MMR.
Why do we need that data?
For one, you could calculate what the "best" build and hero is.
But further, you can find similarities between players, like, calculate how different playerA to others playing Alpha is. Then, you can find out who the closest player to playerA that is performing better is and show the differences.
In simple terms, what do players pick and build that are slightly better than me?
You can always see and talk about the meta as a whole, but you won't have any success with that. Just like picking CP Gwen doesn't help you in T1, WP SAW doesn't help you in T8.
